examples/mui-textfield-integration/RichTextField.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Ref, useRef, useImperativeHandle, useCallback, useEffect, useState } from 'react'
+import TextField, { TextFieldProps } from '@material-ui/core/TextField'
+import { InputBaseComponentProps } from '@material-ui/core/InputBase'
+import { EditorState } from 'draft-js'
+import MUIRichTextEditor from '../../'
+import { TMUIRichTextEditorRef } from '../../src/MUIRichTextEditor'
+
+export interface RichTextInputProps extends Omit<InputBaseComponentProps, 'value'> {
+    inputRef?: Ref<unknown>
+    doFocus?: boolean
+    onStateChange?: (state: EditorState) => void
+}
+
+export const RichTextInput = ({
+    inputRef,
+    doFocus,
+    onStateChange,
+    controls,
+    ...richTextProps
+}: RichTextInputProps) => {
+    const acquireFocus = doFocus ?? false
+
+    // Setup ref for the rich text editor
+    const richTextRef = useRef<TMUIRichTextEditorRef>(null)
+
+    // Attempts to focus the rich text editor reference
+    const focusRichText = useCallback(() => richTextRef.current?.focus(), [richTextRef])
+
+    // Pass on the focus event of the input ref to the rich text ref
+    useImperativeHandle(inputRef, () => ({ focus: () => focusRichText }))
+
+    // If the `acquireFocus` is changed and its value is `true`, focus the editor
+    useEffect(() => {
+        if (acquireFocus) {
+            focusRichText()
+        }
+    }, [acquireFocus, focusRichText])
+
+    return (
+        <MUIRichTextEditor
+            {...richTextProps}
+            ref={richTextRef}
+            onChange={onStateChange}
+        />
+    )
+}
+
+export interface RichTextFieldProps extends Omit<TextFieldProps, 'value' | 'onChange'> {
+    defaultValue?: string
```

Comment:
already in TextFieldProps

Reply:
@TrejGun you're right. I've removed the unnecessary `defaultValue` prop.